src/functions/meta-assistant/send-daily-message/handler.ts

```diff
@@ -16,52 +12,72 @@ import Auth from "src/meta-assistant/auth/auth-provider";
  */
 export const sendDailyMessageHandler = async (): Promise<DailyHandlerResponse> => {
   try {
-    const { accessToken } = await Auth.getAccessToken();
-    if (!accessToken) {
-      throw new Error("Timebank authentication failed");
+    const severaApi = CreateSeveraApiService();
+    const severaUsers = await severaApi.getOptInUsers();
+    const previousWorkDays = TimeUtilities.getPreviousTwoWorkdays();
+    const workHours = await severaApi.getPreviousWorkHours();
+    const slackUsers = await SlackUtilities.getSlackUsers();
+    if (!severaUsers) {
+      throw new Error("No persons retrieved from Severa");
     }
 
-    const previousWorkDays = TimeUtilities.getPreviousTwoWorkdays();
-    const { yesterday, dayBeforeYesterday } = previousWorkDays;
+    // Get the user's guid from workHours
+    const workHoursUserGuids = Array.isArray(workHours) ? workHours.map(hour => hour.user.guid) : [];
```

Comment:
Hopefully this change will also avoid the use of the any type as currently workHoursUserGuids and filteredSeverUsers are being inferred as any which we want to avoid